src/ORM/DB.php

```diff
@@ -550,15 +550,16 @@ public static function create_field($table, $field, $spec)
      * as necessary.
      *
      * @param string $table The name of the table
-     * @param string $fieldSchema A list of the fields to create, in the same form as DataObject::$db
-     * @param string $indexSchema A list of indexes to create.  The keys of the array are the names of the index.
+     * @param array $fieldSchema A list of the fields to create, in the same form as DataObject::$db
+     * @param array $indexSchema A list of indexes to create. See {@link requireIndex()}
```

Comment:
```suggestion
     * @param array|null $fieldSchema A list of the fields to create, in the same form as DataObject::$db
     * @param array|null $indexSchema A list of indexes to create. See {@link requireIndex()}
```
`null` is the default value for these parameters

Reply:
Done

Reply:
Oops, looks like you put the null in the wrong place.